### PR TITLE
fix alerthistory tests

### PR DESCRIPTION
**Pull request by Sild**
no ticket for this issue.
relative to #2981

### CODE REVIEW

**Comment by GregoryKutuzov** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/BaseMethod.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     public static final String MIN_STORABLE_DATE = "1970-01-01T00:00:00.000Z";
     public static final String MAX_STORABLE_DATE = "2106-02-07T06:59:59.999Z";
     public static final String RULE_METRIC_NAME = "test_alert_metric_1";
-    public static final String ALERT_OPEN_VALUE = "0";
+    public static final String ALERT_OPEN_VALUE = "1";
     public static final String ALERT_CLOSE_VALUE = "-1";
     private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/BaseMethod.java]: MethodHandles.lookup().lookupClass()

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/BaseMethod.java]: I believe this is more convenient and clear

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/BaseMethod.java]: fixed in #130

**Comment by rmakulov** [comment]
ok

**Pull request merged**